internal/hcl/evaluator.go

```diff
@@ -516,13 +517,27 @@ func (e *Evaluator) expandBlockForEaches(blocks Blocks) Blocks {
 				cloneValues := clone.Values()
 				e.ctx.Set(cloneValues, append(labels, keyStr)...)
 
-				if v, ok := e.moduleCalls[clone.FullName()]; ok {
-					v.Definition = clone
+				if clone.Type() == "module" {
+					if v, ok := e.moduleCalls[clone.FullName()]; ok {
+						v.Definition = clone
+					} else {
+						modCall, err := e.loadModule(clone)
+						if err != nil {
+							e.logger.WithError(err).Debugf("could not load module %s", clone.FullName())
```

Comment:
can we make this log line a bit different from the last so that we know the error is coming from the for each section. e.g. `failed to create expanded module call, could not load module %s`